Apigee/Mint/DeveloperRatePlan.php

```diff
@@ -395,30 +434,58 @@ public function setDeveloperId($dev)
     }
 
     /**
-     * Convert date string to DateTime object in proper timezone.
+     * Convert date string to DateTime object in organisation's timezone.
```

Comment:
This function should also allow returning date values in a different timezone than org's, but for the backward compatibility, I had to keep it this way.

Possible workaround: introduce new getter callback which accepts destination timezone as a parameter.